### PR TITLE
Integrate Flipper in RNTester on Android

### PR DESCRIPTION
**Pull request by safaiyeh**
## Summary
0.62 Flipper Support items: https://github.com/facebook/react-native/issues/27565
Made RNTester's Android Flipper implementation consistent with the template.
~~Added Flipper to the iOS build.~~
Screen Shot 2019-12-28 at 7 06 42 PM

## Changelog

[Internal] [Added] - RNTester Android Fipper updates

## Test Plan
Run RNTester and see if it connects to Flipper.

### CODE REVIEW

File: RNTester/Podfile [hunk 1]
```diff
@@ -17,6 +52,12 @@ def pods()
   # Additional Pods which aren't included in the default Podfile
   pod 'React-ART', :path => '../Libraries/ART'
   pod 'React-RCTPushNotification', :path => '../Libraries/PushNotificationIOS'
+  
+  # For enabling Flipper.
+  # Note that if you use_frameworks!, flipper will no work.
```
Comment (alloy) [inline RNTester/Podfile]: Why does it not work as framework?

Comment (safaiyeh) [reply RNTester/Podfile]: I do not have enough context to know why. I went off the Flipper docs.

Maybe the Flipper team could give us context.
cc @passy @rickhanlonii @axe-fb

Comment (alloy) [reply RNTester/Podfile]: I’m still naive about Flipper. Could you point me to those docs where it states that?

Comment (safaiyeh) [reply RNTester/Podfile]: Yup! Check out the Podfile example here: https://fbflipper.com/docs/getting-started.html#setup-your-ios-app

**Comment by rickhanlonii** [comment]
Could you rebase and take a look at the failures?

File: RNTester/Podfile (hunk 1 above)
Comment (rickhanlonii) [reply RNTester/Podfile]: Also cc @fkgozali for use_framework implications

Comment (passy) [reply RNTester/Podfile]: I'm a total n00b when it comes to Cocoapods, unfortunately. @priteshrnandgaonkar is our iOS wizard. He's on vacation right now but if this is still a problem next week, I'm sure he can chime in.

**Comment by rickhanlonii** [comment]
I rebased onto master and tested iOS

Screen Shot 2020-01-10 at 11 08 37 AM

**Comment by rickhanlonii** [comment]
Seems like the failing tests are failing on master as well so unrelated

File: RNTester/Podfile (hunk 1 above)
Comment (alloy) [reply RNTester/Podfile]: @safaiyeh Thanks, alas those don’t explain _why_ they need to be build statically. I’ll wait to hear from @fkgozali 👍

Comment (rickhanlonii) [reply RNTester/Podfile]: So Flipper has support for use_frameworks and the instructions above include instructions for use_frameworks, I think we just need to add the right handling and test

Comment (safaiyeh) [reply RNTester/Podfile]: @rickhanlonii I'll try that out today & update the template w/ it.

**Comment by safaiyeh** [comment]
Reverted iOS changes in favor of https://github.com/facebook/react-native/pull/27762, @priteshrnandgaonkar solved the `use_frameworks!` issue.

This just makes the updates to Android to be consistent with the template app

cc @rickhanlonii

**Pull request closed without merging**